### PR TITLE
Extending cheat sheet for pinning

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Cheat Sheet Series. 

Please make sure that for your contribution:

- [ ] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [x] All the markdown files do not raise any validation policy violation, see policy [here](https://github.com/OWASP/CheatSheetSeries#editor--validation-policy).
- [x] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries#conversion-rules).
- [x] All your assets are stored in the **assets** folder.
- [x] All the images used are in the **PNG** format.
- [ ] Any references to website have been formatted as [TEXT](URL)
- [x] You verified/tested the effectiveness of your contribution (e.g.: defensive code proposed is really an effective remediation? Please verify it works!).
- [ ] The CI build of your PR pass, see the build status [here](https://travis-ci.org/OWASP/CheatSheetSeries/pull_requests).

This PR does not follow an issue: it is a out-of-band-attempt to make it more in sync on one of the topics of the MSTG.

Thanks you again for your contribution :smiley:

### CODE REVIEW

**Comment by righettod** [comment]
Thanks for the PR and the reference to the MSTG :)

There no need to add name in the end, we will remove this section at the
end of the year in order to fully use the github contributor system.

Le mar. 11 juin 2019 à 18:58, Jeroen Willemsen <notifications@github.com> a
écrit :

File: cheatsheets/Pinning_Cheat_Sheet.md [hunk 1]
```diff
@@ -82,17 +82,23 @@ To enable pinning, [the `<pin-set>` configuration setting](https://developer.and
...
 
 ## iOS
 
 [TrustKit](https://github.com/datatheorem/TrustKit), an open-source SSL pinning library for iOS and macOS is available. It provides an easy-to-use API for implementing pinning, and has been deployed in many apps.
 
 Otherwise, more details regarding how SSL validation can be customized on iOS (in order to implement pinning) are available in the [HTTPS Server Trust Evaluation](https://developer.apple.com/library/content/technotes/tn2232/_index.html) technical note. However, implementing pinning validation from scratch should be avoided, as implementation mistakes are extremely likely and usually lead to severe vulnerabilities.
 
+Lastly, if you want to validate whether the pinning is succesfull, please follow instructions from the Mobile Security Testing Guide its [introduction into testing network communication](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md#testing-network-communication) and the [iOS specific network testing](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06g-Testing-Network-Communication.md).
```
Comment (ThunderSon) [inline cheatsheets/Pinning_Cheat_Sheet.md]: English mistake: `Testing Guide's [introduction`

File: cheatsheets/Pinning_Cheat_Sheet.md [hunk 2]
```diff
@@ -82,17 +82,23 @@ To enable pinning, [the `<pin-set>` configuration setting](https://developer.and
...
+
+The Android documentation provides an example of how SSL validation can be customized within the app's code (in order to implement pinning) in the [Unknown CA implementation document](https://developer.android.com/training/articles/security-ssl.html#UnknownCa). However, implementing pinning validation from scratch should be avoided, as implementation mistakes are extremely likely and usually lead to severe vulnerabilities.
+
+Lastly, if you want to validate whether the pinning is succesfull, please follow instructions from the Mobile Security Testing Guide its [ntroduction into testing network communication](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md#testing-network-communication) and the [Android specific network testing](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05g-Testing-Network-Communication.md#android-network-apis).
```
Comment (ThunderSon) [inline cheatsheets/Pinning_Cheat_Sheet.md]: English mistake: `Testing Guide's [introduction`

**Comment by commjoen** [comment]
Fixed the english mistakes :). Hope everything is ok now :).

**Pull request merged**